.github/CODEOWNERS

```diff
@@ -2,8 +2,8 @@
 
 # 1. Codeowners must always be teams, never individuals
 # 2. Each codeowner team should contain at least one TSC member
-# 3. PRs touching any code with a codeowner must be signed off by at least one
-#    person on the code owner team.
+# 3. PRs touching any code with a codeowner must tag said owners and wait for an
+#    appropriate time to let them time to review the changes.
```

Comment:
```suggestion
# 3. Landing PRs must follow the rules outlined in
#    https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md#code-reviews
```

Reply:
I don't get the point of linking to the collaborator in this file rather than another one, it has nothing to do with CODEOWNERS. We might as well delete the comment if we think it doesn't bring any value.

Reply:
A `CODEOWNERS` file in general defines rules for changing code under various paths. In our project we’re using it only for automation, rather than enforcing sign-offs, which is how it’s more typically used; so I think it makes sense to say something along the lines of “hey, despite this file’s existence and its typical usage, our policies are as specified over here in this other document.”

We could also delete item 3, sure, but that would leave open the possibility that someone might point to this file as defining our policy. “I had signoff from the team as mentioned in `CODEOWNERS`, it didn’t say I needed anything else, how was I supposed to know there was some other file I needed to look at too?” It doesn’t hurt to be explicit and link, and linking avoids one of these files drifting from the other.

Reply:
What do you think of the following?
```suggestion
# 3. This file does not define any requirements for landing PRs, it's there so
#    the bot can do a courtesy ping to the relevant team(s) when one of the file
#    listed here is touched. Criteria for landing PRs are defined in
#    https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md#code-reviews.
```

Reply:
Sure, here’s with slightly revised grammar:

```suggestion
# 3. This file does not define any requirements for landing PRs.
#    Its purpose is for a GitHub bot to do courtesy pings to the
#    relevant team(s) when any of the paths listed here are modified.
#    Criteria for landing PRs are defined in
#    https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md#code-reviews.
```

Reply:
I would not say "GitHub bot" since the bot has little to do with GitHub.
```suggestion
# 3. This file does not define any requirements for landing PRs.
#    Its purpose is to allow automation of courtesy pings to the
#    relevant team(s) when any of the paths listed here are modified.
#    Criteria for landing PRs are defined in
#    https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md#code-reviews.
```